### PR TITLE
[10.0][ADD] Module project_task_add_very_high

### PR DESCRIPTION
**Pull request by astirpe**
I'm proposing this simple module, since I didn't find any similar module in the community.

### CODE REVIEW

File: project_task_add_very_high/README.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+.. figure:: static/description/image_a.png
+   :alt: On form, priority widget shows one star
+
+and on kanban:
+
+.. figure:: static/description/image2_a.png
+   :alt: On kanban, priority widget shows one star
+
+
+Configuration
```
Comment (elicoidal) [inline project_task_add_very_high/README.rst]: Please remove the empty sections.

**Review by elicoidal — APPROVED** [review]
Very well documented :)

**Comment by astirpe** [comment]
@elicoidal thank you! Done.

**Review by tarteo — APPROVED** [review]
👍 Can you also make project_task_add_ultra_high, please ;) jk

**Comment by dreispt** [comment]
I wish this module did something more, it looks too simple for me.
On the other hand, I don't see any reason to block it.

**Comment by astirpe** [comment]
@dreispt I agree, this module is very simple and that's why I didn't propose it to OCA for a while (it was developed more than one year ago). On the other hand our customers really want it :)

Thanks for merging!

**Pull request merged**